src/utils/index.js

```diff
@@ -1202,3 +1164,7 @@ export const isOctober = () => new Date().getMonth() === 9
 export const isDecember = () => new Date().getMonth() === 11
 
 export { default as totalIngredientsInRecipe } from './totalIngredientsInRecipe'
+export { farmProductsSold } from './farmProductsSold'
+export { isItemAFarmProduct } from './isItemAFarmProduct'
+export { isItemAGrownCrop } from './isItemAGrownCrop'
+export { levelAchieved } from './levelAchieved'
```

Comment:
If it's not too much of a pain, it would be good to keep the code base consistent by removing these exports and updating the downstream `import`s to reference these functions directly. For example: https://github.com/jeremyckahn/farmhand/blob/d96a158ee1d599eaeae296be36962eecd1e6e375/src/data/achievements.js#L12

Reply:
oh, sure! i'm used to still relying on an index file for imports that are outside of the current directory, but i can make that change

Reply:
I think there's a worthwhile discussion to be had about which form is more worth using. On the one hand, direct imports make for fewer layers of indirection and are thus a little easier to follow. On the other hand, index files reduce boilerplate.

For scope of this PR we're better off staying consistent, but I'd be open to revisiting which form this project uses in a dedicated PR. 🙂